tutor4points/core/forms.py

```diff
@@ -235,6 +235,54 @@ def save(self):
         #create and save transaction instance to Transaction table
         Transaction.objects.create (method = 'transfer', points = self.cleaned_data['amount_to_transfer'], sent_from = self.user, sent_to = tutor)
 
+# Form that allows user to send a tutor request
+class TutorRequestForm(forms.ModelForm):
+
+    # def __init__(self, *args,**kwargs):
```

Comment:
You can add an error checking method to the time field, so that the user cannot submit an end time that is the same to earlier than the start time. Refer to the clean_times_available() method in UpdateProfileform for sample.